### PR TITLE
Adds Golden Deep loadout items.

### PR DESCRIPTION
**Pull request by TheStryker**
Adds 19 new Golden Deep loadout items by Noble Row.

lisan al gaib
the peasantry

### CODE REVIEW

**Comment by TheStryker** [comment]
!review

**Comment by Sniblet** [comment]
underscores in player-facing item names
image
eccentric coat has the same thing

File: code/modules/clothing/factions/goldendeep.dm [hunk 1]
```diff
@@ -55,3 +55,174 @@
...
+
+/obj/item/clothing/accessory/goldendeep/gambeson
+	name = "quilted gambeson"
+	desc = " Intricately woven fibers resemble armor of old, but now are reinforced for the wear and tear caused by moving metal joints."
+	icon_state = "quilted_gambeson"
+	item_state = "quilted_gambeson"
+	icon = 'icons/clothing/under/shirts/quilted_gambeson.dmi'
+
+/obj/item/clothing/accessory/goldendeep/tabbard
+	name = "valiant_tabbard"
```
Comment (Ben10083) [inline code/modules/clothing/factions/goldendeep.dm:86]:
```suggestion
	name = "valiant tabbard"
```

File: code/modules/clothing/factions/goldendeep.dm [hunk 2]
```diff
@@ -55,3 +55,174 @@
...
+/obj/item/clothing/accessory/goldendeep/raingarb
+	name = "pactolus rain garb"
+	desc = "Surprisingly sturdy for its patchwork appearance, pieces like this have been protecting workers from Pactolus' radioactive rains for years."
+	icon_state = "pactolus_raingarb"
+	item_state = "pactolus_raingarb"
+	icon = 'icons/clothing/under/shirts/pactolus_raingarb.dmi'
+
+//suits
+/obj/item/clothing/suit/storage/goldendeep
+	name = "eccentric_coat"
```
Comment (Ben10083) [inline code/modules/clothing/factions/goldendeep.dm:115]:
```suggestion
	name = "eccentric coat"
```

**Review by Ben10083 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/factions/goldendeep.dm [hunk 3]
```diff
@@ -55,3 +55,174 @@
...
+	name = "pactolus rain garb"
+	desc = "Surprisingly sturdy for its patchwork appearance, pieces like this have been protecting workers from Pactolus' radioactive rains for years."
+	icon_state = "pactolus_raingarb"
+	item_state = "pactolus_raingarb"
+	icon = 'icons/clothing/under/shirts/pactolus_raingarb.dmi'
+
+//suits
+/obj/item/clothing/suit/storage/goldendeep
+	name = "eccentric coat"
+	desc = " What. Is. That? This coat is long with reinforcement to make someone look like an angel of old, or a clown."
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/goldendeep.dm:116]:
```suggestion
	desc = "This coat is long with reinforcement to make someone look like an angel of old, or a clown."
```

File: code/modules/clothing/factions/goldendeep.dm [hunk 4]
```diff
@@ -55,3 +55,174 @@
...
+	item_state = "eccentric_coat"
+	icon = 'icons/clothing/suits/coats/eccentric_coat.dmi'
+	contained_sprite = TRUE
+	has_accents = TRUE
+
+//cloaks
+
+/obj/item/clothing/accessory/poncho/goldendeep
+	name = "golden deep cape"
+	desc = "Viewed as ostentatious by outsiders (and perhaps insiders, for that matter,) capes nonetheless remain as a means of quickly broadcasting one's status (or pretentions of status)."
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/goldendeep.dm:127]:
```suggestion
	desc = "Viewed as ostentatious by outsiders (and perhaps insiders, for that matter) capes nonetheless remain as a means of quickly broadcasting one's status (or pretentions of status)."
```

File: code/modules/clothing/factions/goldendeep.dm [hunk 5]
```diff
@@ -55,3 +55,174 @@
...
+	name = "flowing cloak"
+	desc = "A long and flowing cloak used by members of Golden Deep evocative of trenchcoats. Curiously, many investigatorial synthetics within the collective favor these cloaks."
+	icon_state = "flowing_cloak"
+	item_state = "flowing_cloak"
+	icon = 'icons/clothing/suits/capes/flowing_cloak.dmi'
+	has_accents = TRUE
+
+/obj/item/clothing/accessory/poncho/goldendeep/elegantcloak
+	name = "elegant cloak"
+	desc = " Billowing fabric for all classes with a regal clasp that hints at its original merchant designers."
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/goldendeep.dm:144]:
```suggestion
	desc = "Billowing fabric for all classes with a regal clasp that hints at its original merchant designers."
```

File: code/modules/clothing/factions/goldendeep.dm [hunk 6]
```diff
@@ -55,3 +55,174 @@
...
+	icon_state = "elegant_cloak"
+	item_state = "elegant_cloak"
+	icon = 'icons/clothing/suits/capes/elegant_cloak.dmi'
+	has_accents = TRUE
+
+/obj/item/clothing/accessory/poncho/goldendeep/furwrap
+	name = "fur wrap"
+	desc = "The fur of an animal, soft and fluffy, now repurposed as a decorative neck wrap. "
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/goldendeep.dm:152]:
```suggestion
	desc = "The fur of an animal, soft and fluffy, now repurposed as a decorative neck wrap."
```

File: code/modules/clothing/factions/goldendeep.dm [hunk 7]
```diff
@@ -55,3 +55,174 @@
...
+	desc = "Lightweight and soft, this sleeveless coat feels like it's hardly anything at all, but it keeps the sun from bearing down on the wearer."
+	icon_state = "desert_adornment"
+	item_state = "desert_adornment"
+	icon = 'icons/clothing/suits/capes/desert_adornment.dmi'
+
+//head slot
+
+/obj/item/clothing/head/goldendeep
+	name = "furred crown"
+	desc = "Like a true monarch, has a metal rim capped with a cloth top piece. Peaked in a ornate point. "
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/goldendeep.dm:168]:
```suggestion
	desc = "Like a true monarch, has a metal rim capped with a cloth top piece. Peaked in a ornate point."
```

File: code/modules/clothing/factions/goldendeep.dm [hunk 8]
```diff
@@ -55,3 +55,174 @@
...
+	slot_flags = SLOT_EARS  | SLOT_HEAD
+
+/obj/item/clothing/head/goldendeep/cubehood
+	name = "cube hood"
+	desc = "Ornate and with subtle religious undertones, would make any wearer loom like a revered parent."
+	icon_state = "cube_hood"
+	item_state = "cube_hood"
+	icon = 'icons/clothing/head/cube_hood.dmi'
+	body_parts_covered = 0
+	slot_flags = SLOT_EARS  | SLOT_HEAD
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/goldendeep.dm:191]:
```suggestion
	slot_flags = SLOT_EARS | SLOT_HEAD
```

File: code/modules/clothing/factions/goldendeep.dm [hunk 9]
```diff
@@ -55,3 +55,174 @@
...
+
+/obj/item/clothing/head/goldendeep/chainjewelry
+	name = "chain jewelry"
+	desc = "Jingling and dancing with movement. This chain is of metals or jewels that flaunt the wealth of the wearer. Or some facsimile of it."
+	icon_state = "chain_jewelry"
+	item_state = "chain_jewelry"
+	icon = 'icons/clothing/head/chain_jewelry.dmi'
+	slot_flags = SLOT_EARS  | SLOT_HEAD | SLOT_MASK
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/goldendeep.dm:199]:
```suggestion
	slot_flags = SLOT_EARS | SLOT_HEAD | SLOT_MASK
```

File: code/modules/clothing/factions/goldendeep.dm [hunk 10]
```diff
@@ -55,3 +55,174 @@
...
+
+/obj/item/clothing/head/goldendeep/chainjewelry/baseline
+	icon_state = "chain_jewelry_baseline"
+	item_state = "chain_jewelry_baseline"
+
+//accessories
+
+/obj/item/clothing/accessory/storage/goldendeep
+	name = "thesian webbing"
+	desc = " For the hard working Thesian of the Golden Deep. Able to carry tools and withstand the rigors of work within the collective."
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/goldendeep.dm:209]:
```suggestion
	desc = "For the hard working Thesian of the Golden Deep. Able to carry tools and withstand the rigors of work within the collective."
```

**Review by SleepyGemmy — COMMENTED** [review]
**general**
- underscores in paths (i.e. `golden_deep` instead of `goldendeep`) is preferable.
- icon definition should preferably go before icon and item states.

**Review by Geevies — APPROVED** [review]
code looks good

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**